test/test_helper.rb

```diff
@@ -265,6 +265,18 @@ class ParanoidTime < ActiveRecord::Base
   has_one :has_one_not_paranoid, dependent: :destroy
 
   belongs_to :not_paranoid, dependent: :destroy
+
+  attr_accessor :destroyable
+
+  before_destroy :ensure_destroyable
+
+  protected
+
+  def ensure_destroyable
+    return if destroyable.nil?
```

Comment:
This line is not needed since `nil` is falsey.

Reply:
this is to skip the check of `destroyable` if it's not set

Reply:
Ah yes, of course. I got the logic reversed in my head.